tests/integration_tests/reports/commands/execute_dashboard_report_tests.py

```diff
@@ -66,3 +67,47 @@ def test_report_for_dashboard_with_tabs(
         assert digest == dashboard.digest
         assert send_email_smtp_mock.call_count == 1
         assert len(send_email_smtp_mock.call_args.kwargs["images"]) == 1
+
+
```

Comment:
consider adding unit tests for `send_email_smtp` with the actual use of `NOTIFICATION_EMAIL_HEADER_MUTATOR`

Reply:
Would that exist in this repo or in one with an actual `NOTIFICATION_EMAIL_HEADER_MUTATOR`?

Reply:
You can mock an example `NOTIFICATION_EMAIL_HEADER_MUTATOR` and test it in this repo